src/box/lua/iproto.c

```diff
@@ -119,6 +139,17 @@ push_iproto_key_enum(struct lua_State *L)
 		{"VCLOCK_SYNC", IPROTO_VCLOCK_SYNC},
 	};
 	push_iproto_constant_subnamespace(L, "key", keys, lengthof(keys));
+	for (size_t i = 0; i < lengthof(keys); ++i) {
+		size_t len = strlen(keys[i].name);
+		const char *name = to_lower_copy(keys[i].name, len);
```

Comment:
Shouldn't upper-case variants be translated as well?

Reply:
Yeah, I guess so, added translation of upper-case constants too.

I also decided to make the IPROTO constant arrays and aliases static.